index.bs

```diff
@@ -788,19 +787,19 @@ Similarly, when piping a {{ReadableStream}} into a {{FileSystemWritableFileStrea
 
 <div algorithm>
 To <dfn>create a new FileSystemWritableFileStream</dfn> given a [=file entry=] |file|
-in a [=/Realm=] |realm|, perform the following steps:
+in a [=/Realm=] |realm|, run these steps:
 
 1. Let |stream| be a [=new=] {{FileSystemWritableFileStream}} in |realm|.
 1. Set |stream|.[=FileSystemWritableFileStream/[[file]]=] to |file|.
 1. Let |writeAlgorithm| be an algorithm which takes a |chunk| argument
    and returns the result of running the [=write a chunk=] algorithm with |stream| and |chunk|.
-1. Let |closeAlgorithm| be the following steps:
+1. Let |closeAlgorithm| be these steps:
    1. Let |closeResult| be [=a new promise=].
-   1. Run the following steps [=in parallel=]:
+   1. Run these steps [=in parallel=]:
       1. Let |access| be the result of running |file|'s [=entry/query access=] given "`readwrite`".
       1. If |access| is not "{{PermissionState/granted}}",
          reject |closeResult| with a {{NotAllowedError}} and abort.
-      1. Perform [=implementation-defined=] malware scans and safe browsing checks.
+      1. Run [=implementation-defined=] malware scans and safe browsing checks.
```

Comment:
Note: this change is not find-and-replace, but it seems to fit the pattern

Reply:
Yeah. I'm not sure how good we are on "Run" vs "Perform" in general. Both seem okay and JavaScript uses "Perform" a lot, but the web platform has indeed more "Run".